guides/deployments/podman.md

```diff
@@ -137,69 +148,85 @@ Linux kernel doesn't support CVMs.
    ```yaml
    version: "0.2"
    workspace:
-       configure:
-           start:
-               policy: write
-       dev-urls:
+     configure:
+       start:
+         policy: write
+     dev-urls:
+       policy: write
+     specs:
+       aws-ec2-docker:
+         container-image:
+           policy: write
+         disk-size:
+           policy: write
+         instance-type:
+           policy: write
+       docker:
+         container-based-vm:
+           policy: write
+         image:
+           policy: write
+       kubernetes:
+         annotations:
+           policy: read
+         container-based-vm:
+           policy: write
+         cpu:
            policy: write
-       specs:
-           aws-ec2-docker:
-               container-image:
-                   policy: write
-               disk-size:
-                   policy: write
-               instance-type:
-                   policy: write
-           docker:
-               container-based-vm:
-                   policy: write
-               image:
-                   policy: write
-           kubernetes:
-               annotations:
-                   policy: read
-               container-based-vm:
-                   policy: write
-               cpu:
-                   policy: write
-               disk:
-                   policy: write
-               env:
-                   policy: write
-               gpu-count:
-                   policy: write
-               image:
-                   policy: write
-               labels:
-                   policy: read
-               memory:
-                   policy: write
-               node-selector:
-                   policy: read
-               privileged:
-                   policy: read
-               resource-requests:
-                policy: write
-                value:
-                   smarter-devices/fuse: "1"
-               resource-limits:
-                policy: write
-                value:
-                   smarter-devices/fuse: "1"
-               runtime-class-name:
-                   policy: read
-               tolerations:
-                   policy: read
+         disk:
+           policy: write
+         env:
+           policy: write
+         gpu-count:
+           policy: write
+         image:
+           policy: write
+         labels:
+           policy: read
+         memory:
+           policy: write
+         node-selector:
+           policy: read
+         privileged:
+           policy: read
+         resource-requests:
+           policy: write
+           value:
+             smarter-devices/fuse: "1"
+         resource-limits:
+           policy: write
+           value:
+             smarter-devices/fuse: "1"
```

Comment:
These lines are really the only relevant ones, so if we had some way to highlight these (to indicate to users that this is what is needed to mount the FUSE device needed to run Podman), then that would clarify what this template is showing